Sources/Danger/Danger.swift

```diff
@@ -65,72 +71,142 @@ public func Danger() -> DangerDSL {
     return DangerRunner.shared.dsl
 }
 
+extension DangerDSL {
+    /// Fails on the Danger report
+    public var fails: [Violation] {
```

Comment:
I added this to allow people that are using multiple files to use danger to post messages etc. with typing and auto complete, without have to `import Danger` in all the files (given you have a global instance of danger defined on the Dangerfile, and that the multiple files will be copied on the Dangerfile when danger is executed)
(It works also with the import anyway)

But in this way you get 
<img width="781" alt="screenshot 2018-11-05 at 19 48 07" src="https://user-images.githubusercontent.com/17830956/48024207-f2566800-e137-11e8-8177-a450d2277f74.png">
without import Danger